<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMessageWindow</class>
 <widget class="QWidget" name="NewMessageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#NewMessageWindow{background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:0.503, y2:0.306727, stop:0.0696517 rgba(169, 169, 169, 255), stop:0.890547 rgba(255, 255, 255, 255));}
#stackedWidget{ background-color: rgba(1,1,1,0);}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="NewMessageTitle">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New Message</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="NWTopicLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Topic:</string>
     </property>
    </widget>
    <widget class="QLabel" name="NWLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NWBackButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>354</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NWClearButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>354</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NWSendButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>354</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>138</red>
           <green>226</green>
           <blue>52</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>138</red>
           <green>226</green>
           <blue>52</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>138</red>
           <green>226</green>
           <blue>52</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NWTopic">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NWInputBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="NWChatLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Chat:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NWSubscribeButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>354</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Subscribe</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="NWChatBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="NWSendURL">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>150</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Link</string>
     </property>
    </widget>
    <widget class="QLabel" name="NWURLLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>177</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>URL will appear here when sent</string>
     </property>
    </widget>
    <widget class="QLabel" name="NWFileLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>File Link:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NWFileLinkInput">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="NWDisconnect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AnjaliOldLipi</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NWEmoji">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Emoji</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
